### DISCUSSION TITLE
Can we configure GitHub hosted App?

### DISCUSSION THREAD

**Discussion by latipun7** [renovatebot/renovate#11262] [Request Help] [answered]
Can we configure [self-hosted configuration](https://docs.renovatebot.com/self-hosted-configuration/) for the [GitHub hosted App](https://github.com/marketplace/renovate)? For example in the repo `.github/config.yml`

Especially I want to update `allowPostUpgradeCommandTemplating` and `allowedPostUpgradeCommands` configs, since I think GitHub Hosted App didn't enable them(?).

Thank you.

**Comment by rarkins** [comment, marked as answer]
No, you can't
